tests/unittests/OperatorTest.cpp

```diff
@@ -275,24 +283,22 @@ TEST_P(InterpAndCPU, batchedReduceAddQuantizedWithAxis) {
   auto BT = mod_.uniqueType(ElemKind::Int8QTy, {2, 3, 4}, 0.5, 3);
   auto OT = mod_.uniqueType(ElemKind::Int8QTy, {2, 4}, 2.0, -1);
 
-  auto *batch = mod_.createVariable(ElemKind::Int8QTy, {2, 3, 4},
-                                    BT->getScale(), BT->getOffset(), "batch");
-  auto *result = mod_.createVariable(ElemKind::Int8QTy, {2, 4}, OT->getScale(),
-                                     OT->getOffset(), "result");
+  auto *batch =
+      mod_.createPlaceholder(ElemKind::Int8QTy, {2, 3, 4}, BT->getScale(),
+                             BT->getOffset(), "batch", false);
 
-  batch->getPayload().getHandle<int8_t>() = {
+  ctx_.allocate(batch)->getHandle<int8_t>() = {
       27, -31, 16,  7,  20, 34, -2, 8,   -10, 83, 29,  -17,
       19, 13,  -11, -9, 50, 58, 0,  -20, -72, 43, -25, -1};
 
-  auto BH = batch->getHandle<int8_t>();
-  auto OH = result->getHandle<int8_t>();
+  auto BH = ctx_.allocate(batch)->getHandle<int8_t>();
```

Comment:
Should be ctx_.get(batch), since batch has been allocated in line 290

Reply:
Indeed!